### DISCUSSION TITLE
how to make vlc the default player?

### DISCUSSION THREAD

**Discussion by cristian158** [pystardust/ani-cli#550] [General] [answered]
just instead of using mpv

**Comment by port19x** [comment, marked as answer]
If you're on linux or mac you can set an alias.
`alias ani-cli="ani-cli -v"`
I'm sure windows should have something like aliases too.

**Reply by cristian158** [reply]
Linux
Did alias on .zshrc, nothing.
Even when I run it with -v it outputs "Program "mpv" not found. Please install it."
